src/adapters/RoomsAdapter.js

```diff
@@ -56,7 +61,7 @@ export default class RoomsAdapter extends WebexAdapter {
    * Returns an observable that emits an array of current activity IDs of the given roomID.
```

Comment:
Since we're returning more than IDs now:
```suggestion
   * Returns an observable that emits an array of current activities of the given roomID.
```